test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
```

Comment:
All the tests are run with `-net none` and some of the tests mask `systemd-networkd` explicitly. I wonder why `systemd.mask=systemd-networkd.service` is needed?